Typescript issue when module resolution Node16 or Bundler used

Thank you for the great library, I've been using it fine with my current `moduleResolution` option of "node" however other libraries I'm using require me to change to a more modern option for typescript to find the types.

If I change my project (vue + vite) to use the default or "NodeNext" module resolution I get this error:
```
error TS7016: Could not find a declaration file for module '@placemarkio/tokml'. '.../node_modules/@placemarkio/tokml/dist/tokml.es.mjs' implicitly has an 'any' type.
  There are types at '.../node_modules/@placemarkio/tokml/dist/index.d.ts', but this result could not be resolved when respecting package.json "exports". The '@placemarkio/tokml' library may need to update its package.json or typings.

1 import { toKML } from '@placemarkio/tokml';
```
This is the related TS issue: https://github.com/microsoft/TypeScript/issues/52363

There is a tool here that allows you to check typings on a published NPM repo here: https://arethetypeswrong.github.io/?p=%40placemarkio%2Ftokml%400.3.3

If I add this line to the package.json for this library manually in my local node_modules the error goes away:
![Screenshot 2024-09-09 at 2 25 29 PM](https://github.com/user-attachments/assets/bce24857-194d-48f3-a3f2-90e38acfd2e1)

--- a/package.json
+++ b/package.json
@@ -1,61 +1,57 @@
 {
-  "name": "@placemarkio/tokml",
-  "version": "0.3.3",
-  "description": "Convert GeoJSON to KML",
-  "main": "dist/tokml.cjs",
-  "module": "dist/tokml.es.mjs",
-  "typings": "dist/index.d.ts",
-  "author": "Tom MacWright",
-  "license": "MIT",
-  "exports": {
-    "require": "./dist/tokml.cjs",
-    "default": "./dist/tokml.es.mjs"
-  },
-  "keywords": [
-    "kml",
-    "geojson"
-  ],
-  "publishConfig": {
-    "access": "public"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "yarn lint"
-    }
-  },
-  "prettier": {
-    "printWidth": 80,
-    "semi": true,
-    "singleQuote": true,
-    "trailingComma": "es5"
-  },
-  "files": [
-    "dist"
-  ],
-  "scripts": {
-    "test": "vitest",
-    "lint": "eslint . && tsc --noEmit",
-    "doc": "typedoc --options typedoc.json",
-    "prepare": "rollup -c rollup.config.mjs && npm run doc && cp CNAME docs/CNAME",
-    "release": "standard-version"
-  },
-  "devDependencies": {
-    "@rollup/plugin-node-resolve": "^15.0.1",
-    "@rollup/plugin-typescript": "^9.0.2",
-    "@tmcw/togeojson": "5.4.0",
-    "@types/geojson": "^7946.0.8",
-    "eslint": "^8.18.0",
-    "prettier": "^2.7.1",
-    "rollup": "^3.2.4",
-    "rollup-plugin-dts": "^5.0.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "standard-version": "^9.5.0",
-    "tslib": "^2.4.0",
-    "typedoc": "^0.23.19",
-    "typescript": "^4.7.4",
-    "unist-builder": "^3.0.0",
-    "vitest": "^0.24.4",
-    "xast-util-to-xml": "^3.0.0",
-    "xastscript": "^3.0.0"
-  }
+	"name": "@placemarkio/tokml",
+	"version": "0.3.3",
+	"description": "Convert GeoJSON to KML",
+	"main": "dist/tokml.cjs",
+	"module": "dist/tokml.es.mjs",
+	"typings": "dist/index.d.ts",
+	"author": "Tom MacWright",
+	"license": "MIT",
+	"exports": {
+		"require": "./dist/tokml.cjs",
+		"default": "./dist/tokml.es.mjs",
+		"types": "./dist/index.d.ts"
+	},
+	"keywords": ["kml", "geojson"],
+	"publishConfig": {
+		"access": "public"
+	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "yarn lint"
+		}
+	},
+	"prettier": {
+		"printWidth": 80,
+		"semi": true,
+		"singleQuote": true,
+		"trailingComma": "es5"
+	},
+	"files": ["dist"],
+	"scripts": {
+		"test": "vitest",
+		"lint": "eslint . && tsc --noEmit",
+		"doc": "typedoc --options typedoc.json",
+		"prepare": "rollup -c rollup.config.mjs && npm run doc && cp CNAME docs/CNAME",
+		"release": "standard-version"
+	},
+	"devDependencies": {
+		"@rollup/plugin-node-resolve": "^15.0.1",
+		"@rollup/plugin-typescript": "^9.0.2",
+		"@tmcw/togeojson": "5.4.0",
+		"@types/geojson": "^7946.0.8",
+		"eslint": "^8.18.0",
+		"prettier": "^2.7.1",
+		"rollup": "^3.2.4",
+		"rollup-plugin-dts": "^5.0.0",
+		"rollup-plugin-terser": "^7.0.2",
+		"standard-version": "^9.5.0",
+		"tslib": "^2.4.0",
+		"typedoc": "^0.23.19",
+		"typescript": "^4.7.4",
+		"unist-builder": "^3.0.0",
+		"vitest": "^0.24.4",
+		"xast-util-to-xml": "^3.0.0",
+		"xastscript": "^3.0.0"
+	}
 }

